TopsortAnalytics/src/main/java/com/topsort/analytics/service/TopsortAnalyticsHttpService.kt

```diff
@@ -0,0 +1,49 @@
+package com.topsort.analytics.service
+
+import com.google.gson.Gson
+import com.topsort.analytics.Cache
+import com.topsort.analytics.core.HttpClient
+import com.topsort.analytics.core.HttpResponse
+import com.topsort.analytics.model.ClickEvent
+import com.topsort.analytics.model.ImpressionEvent
+import com.topsort.analytics.model.PurchaseEvent
+
+private const val apiUrl = "https://api.topsort.com/v1/events"
```

Comment:
Can the hostname be a project wide constant and potentially overridable? Lets say we wanna point it at a staging or test environment

Reply:
I'll address in future PRs when we get a better testing setup